src/component/App.js

```diff
@@ -1,12 +1,46 @@
 import './App.css';
-import DataTable from './DataTable'
+
+import firebase from 'firebase/app';
+import 'firebase/firestore';
+import 'firebase/auth';
+
+import { useCollectionData } from 'react-firebase-hooks/firestore';
+
+firebase.initializeApp({
+    apiKey: "AIzaSyBe9RfBFcXjrGtQ7jVxtrxSOWhqf28DHYk",
+    authDomain: "gwwc-data-table.firebaseapp.com",
+    databaseURL: "https://gwwc-data-table.firebaseio.com",
+    projectId: "gwwc-data-table",
+    storageBucket: "gwwc-data-table.appspot.com",
+    messagingSenderId: "924988001345",
+    appId: "1:924988001345:web:655254cb48b7162b6f4ac0",
+    measurementId: "G-TC4E5JES7F"
+})
+
+const firestore = firebase.firestore();
 
 function App() {
+
   return (
     <div className="App">
+      <p>here's the data table:</p>
       <DataTable></DataTable>
```

Comment:
We can pass data  to the `DataTable` component by loading it in the app render function as you do below and then passing an `organizations` variables to the child `DataTable` as:

`<DataTable organizations={organizations}></DataTable>`.

In the `DataTable.js` file this can be accessed as 

```
function DataTable(props) {
    const data = props.organizations;

    ....
}
```

We can pass this to the child component by any name so `<DataTable data={organizations}></DataTable>` could be accessed as `props.data` and would include the same info. See [here](https://reactjs.org/docs/components-and-props.html) for more info on props